src/App.js

```diff
@@ -1,12 +1,63 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
 import './App.css';
+import axios from "axios";
+import PokemonFiche from "./components/PokemonFiche";
+import Pagination from "./components/Pagination";
+import PokemonBall from './assets/logos/pokemon-pokeball-vector-graphic.png'
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+    const [pokemonArray, setPokemonArray] = useState([]);
+    const [currentPage, setCurrentPage] = useState("https://pokeapi.co/api/v2/pokemon")
+    const [nextPage, setNextPage] = useState("");
+    const [previousPage, setPreviousPage] = useState("");
+    const [loading, setLoading] = useState(true); /*our application is loading! Within this we need info for user */
+
+
+    useEffect(() => {
+        async function fetchEmAll() {
+
+            try {
+                setLoading(true);
+                const response = await axios.get(currentPage);
```

Comment:
Hier zou je nog spreading kunnen toepassen
```suggestion
                const { data: { results, next, previous } } = await axios.get(currentPage);
```